### ISSUE TITLE
appraisal:install doesn't work if bundler installs gems inside the project

### ISSUE THREAD

**Issue by jcoglan** [thoughtbot/appraisal#55]
Steps to reproduce:

```
$ git clone https://gist.github.com/6130471.git
$ cd 6130471/
$ bundle --path .bundle
$ bundle exec rake appraisal:install
>> bundle check --gemfile='/home/james/projects/tmp/6130471/gemfiles/v1.gemfile' || bundle install --gemfile='/home/james/projects/tmp/6130471/gemfiles/v1.gemfile'
/opt/rubies/1.9.3-p448/lib/ruby/1.9.1/rubygems/dependency.rb:247:in `to_specs': Could not find bundler (>= 0) amongst [appraisal-0.5.2, rake-10.1.0] (Gem::LoadError)
    from /opt/rubies/1.9.3-p448/lib/ruby/1.9.1/rubygems/dependency.rb:256:in `to_spec'
    from /opt/rubies/1.9.3-p448/lib/ruby/1.9.1/rubygems.rb:1231:in `gem'
    from /opt/rubies/1.9.3-p448/bin/bundle:22:in `<main>'
```

I can only get Appraisal to work if I bundle my project's deps into the global gem installation space (either in `~/.gem` or in the Ruby installation in `/opt/rubies`).

**Comment by joshuap** [comment]
I was able to fix this issue on CircleCI by installing bundler into my vendor/bundle path before running `bundle exec appraisal install`: `gem install bundler --install-dir=/home/ubuntu/honeybadger-ruby/vendor/bundle/ruby/1.9.1`

**Comment by jferris** [comment]
@sikachu if you fork and load the Bundler CLI in Ruby (instead of shelling out), bundler will be available, right?

**Comment by sikachu** [comment]
@jferris I haven't tried that yet. I'll give that a try tomorrow and report back.

**Comment by Senjai** [comment]
@sikachu Any news on this one?

**Comment by sikachu** [comment]
@Senjai I fixed it before 2.0.0 release, so it should be out with Appraisal 2.0.0. If you still seeing the issue, please let me know.
